notifications_utils/sanitise_text.py

```diff
@@ -96,9 +100,11 @@ class SanitiseGSM(SanitiseText):
     """
     ALLOWED_CHARACTERS = set(
         '@£$¥èéùìòÇ\nØø\rÅåΔ_ΦΓΛΩΠΨΣΘΞ\x1bÆæßÉ !"#¤%&\'()*+,-./0123456789:;<=>?' +
-        '¡ABCDEFGHIJKLMNOPQRSTUVWXYZÄÖÑÜ`¿abcdefghijklmnopqrstuvwxyzäöñüà' +
+        '¡ABCDEFGHIJKLMNOPQRSTUVWXYZÄÖÑÜ§¿abcdefghijklmnopqrstuvwxyzäöñüà' +
```

Comment:
any reason `` ` `` has been replaced with `§`?

Reply:
All the GSM character lists I came across have `§` in instead of `` ` `` (there are no other differences between our list and the ones I found online). So I guessed that we had a small mistake in our list?